src/sa_web/static/libs/leaflet-mapbox-gl.js

```diff
@@ -1,100 +0,0 @@
-L.MapboxGL = L.Class.extend({
```

Comment:
I don't think we want to delete this file. Sorry if I said that we should delete it. I think we just want to leave this file alone and not have any diffs on it.

But here is a file that we should delete: https://github.com/smartercleanup/platform/pull/254/files#r51521140